editor/src/clj/editor/properties_view.clj

```diff
@@ -282,9 +292,9 @@
                 children (if (seq label-text)
                            (let [label (doto (Label. label-text)
                                          (.setMinWidth Region/USE_PREF_SIZE))]
-                             (ui/do-run-later
-                               #(when-not (properties/read-only? (property-fn))
-                                  (make-label-draggable! label (partial handle-label-drag-event! property-fn drag-update-fn update-ui-fn))))
+                             (ui/run-later
+                               (when-not (properties/read-only? (property-fn))
+                                 (make-label-draggable! label (partial handle-label-drag-event! property-fn drag-update-fn update-ui-fn))))
```

Comment:
Unrelated to the issue. This has the same effect, but I think I should have used `ui/run-later` in the first place.